### PR TITLE
[MIG] sale_commission_salesman: Migration to 17.0

### PR DESCRIPTION
**Pull request by CLaurelB**
Changelog:
- Replace `attrs` with an expression for the `invisible` attribute in the view.

### CODE REVIEW

**Comment by CLaurelB** [comment]
@luisg123v could you review, please?

File: sale_commission_salesman/README.rst [hunk 1]
```diff
@@ -17,23 +17,24 @@ Sales commissions from salesman
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcommission-lightgray.png?logo=github
-    :target: https://github.com/OCA/commission/tree/16.0/sale_commission_salesman
+    :target: https://github.com/OCA/commission/tree/17.0/sale_commission_salesman
```
Comment (luisg123v) [inline sale_commission_salesman/README.rst:20]: Commit title:

execution → autofixes

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: sale_commission_salesman/README.rst (hunk 1 above)
Comment (CLaurelB) [reply sale_commission_salesman/README.rst:20]: Title updated.

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

@moylop260 could you review/merge, please?

Regards

CC @desdelinux

**Pull request merged**